src/sentry/sentry_metrics/use_case_id_registry.py

```diff
@@ -25,6 +27,12 @@ class UseCaseID(Enum):
     UseCaseKey.PERFORMANCE: UseCaseID.TRANSACTIONS,
 }
 
+USE_CASE_ID_WRITES_LIMIT_QUOTA_OPTION_NAME = {
+    UseCaseID.SPANS: "sentry-metrics.writes-limiter.limits.spans",
```

Comment:
Since we added the spans use case here in order to test out the write limiter functionality, could we maybe use a mock/test UseCaseID instead of spans?